sys-libs/gdbm/gdbm-1.23.ebuild

```diff
@@ -20,6 +20,11 @@ DEPEND="readline? ( sys-libs/readline:=[${MULTILIB_USEDEP}] )"
 RDEPEND="${DEPEND}"
 BDEPEND="verify-sig? ( sec-keys/openpgp-keys-gdbm )"
 
+src_prepare() {
+	default
+	elibtoolize
```

Comment:
Let's add a comment explaing why to avoid this happening in future.

Do wonder if it's stale in the tarball or if the fix never actually landed in upstream libtool so we need our patches for it.

Reply:
Ah, checking `build-aux/ltmain.sh`:
```
#! /bin/sh
## DO NOT EDIT - This file generated from ./build-aux/ltmain.in
##               by inline-source v2014-01-03.01

# libtool (GNU libtool) 2.4.6
# Provide generalized library-building support services.
# Written by Gordon Matzigkeit <gord@gnu.ai.mit.edu>, 1996

# Copyright (C) 1996-2015 Free Software Foundation, Inc.
# This is free software; see the source for copying conditions.  There is NO
# warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
[...]
```

so yeah, their distro hasn't got 2.4.7 yet, and even that may not have the needed patches.